gulpfile.js

```diff
@@ -4,7 +4,19 @@ var gulp = require('gulp'),
 	nodemon = require('gulp-nodemon'),
 	browserSync = require('browser-sync'),
 	stylus = require('gulp-stylus'),
-	connect = require ('gulp-connect');
+	connect = require ('gulp-connect'),
+	connect         = require('gulp-connect'),
+    // mocha           = require('gulp-mocha'),
```

Comment:
For release, remove all commented out code that does not provide additional functionality.